packages/playback/src/lib/types/eme-manager.declarations.ts

```diff
@@ -16,3 +19,18 @@ export interface IEmeManager {
   handleWaitingForKey(): void;
   setInitData(type: string, data: ArrayBuffer): void;
 }
+
+export interface IEmeApiAdapter {
+  // TODO: implement this adapter type
+  id: string;
+}
+
+export interface IKeySystemConfiguration {
+  label?: string;
+  initDataTypes?: Array<string>;
+  audioCapabilities?: Array<MediaKeySystemMediaCapability>;
+  videoCapabilities?: Array<MediaKeySystemMediaCapability>;
+  distinctiveIdentifier?: MediaKeysRequirement;
+  persistentState?: MediaKeysRequirement;
+  sessionTypes?: Array<string>;
+}
```

Comment:
you do not have to declare it, it should be in the built-in typescript types for browser